app/i18n/de/install.php

```diff
@@ -4,7 +4,9 @@
 	'action' => array(
 		'finish' => 'Installation fertigstellen',
 		'fix_errors_before' => 'Bitte Fehler korrigieren, bevor zum nächsten Schritt gesprungen wird.',
+		'keep_install' => 'Keep previous installation',  // TODO: translate
```

Comment:
Keep previous installation --> Vorherige Installation beibehalten (Daten)
Reinstall FreshRSS --> Neuinstallation von FreshRSS
We have detected that FreshRSS is already installed! --> Wir haben festgestellt, dass FreshRSS bereits installiert wurde!